### ISSUE TITLE
Some improvements

### ISSUE THREAD

**Issue by obiwankennedy** [codestation/python-phpbb3#2]
Hi! 

thanks for your project it helps a lot.
I edit the isLogged function to adapt to the cookies name, i get. But it may cause other issues. 

```python
def isLogged(self):
        if self.jar != None:
            for cookie in self.jar:
                if re.search('.*_u', cookie.name) and cookie.value:
                    return True
        return False
```

A way to set the cookie.name patterns may fix this issue.

**Comment by codestation** [comment]
I put the cookie pattern as a class variable so you can override it. Don't expect more updates since i moved long ago from being a forum moderator (also i would pick Go over python nowadays).

Fixed by 1687643abc7065e88be0e446078cc14eecb37340

**Comment by obiwankennedy** [comment]
Even with the pattern change I can't log on some phpBB. 
I change  completely the way I will do that. I create a webbrowser in C++, and it is done.
